README.md

```diff
@@ -17,4 +17,8 @@ The figma designs can be found [here](https://www.figma.com/file/xsPf6IIM9AevLN5
 
 ## Sequence diagram
 
-[![](docs/images/architecture.png)](https://docs.google.com/presentation/d/16038cTBefSKQezJk0IZKNXnSqaG2PnU07Sb2_qIkNe8/edit?usp=sharing)
\ No newline at end of file
+[![](docs/images/architecture.png)](https://docs.google.com/presentation/d/16038cTBefSKQezJk0IZKNXnSqaG2PnU07Sb2_qIkNe8/edit?usp=sharing)
+
+## Node version
+
+Optic uses the current LTS version of Node
```

Comment:
let's move this to the top, it's one of the first things that somebody who needs to run the app locally will need to know. Let's create a Requirements heading and list this and yarn as requirements